src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -1,16 +1,27 @@
 package mate.academy.service;
 
+import mate.academy.model.User;
+
 public class AuthenticationService {
+    private UserService userService = new UserService();
+
     /**
      * Imagine that some user wants to login to your site.
      * You should check if user credentials (login and password) are valid or not.
      * All users are stored in <code>UserService</code> class.
-     * @param email - user's email
+     *
+     * @param email    - user's email
      * @param password - user's password
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+        User user = userService.findByEmail(email);
+        if (user != null
+                && user.getEmail() == email
+                && user.getPassword() == password) {
```

Comment:
no reason to check Email here
do not compare Strings using '==' - use **equals**
try to do it without any `if` in this method

Reply:
thanks I got it!)... hope)